src/components/NavigationDesktop/NavigationItemDesktop.js

```diff
@@ -90,10 +90,10 @@ class NavigationItemDesktop extends Component {
   };
 
   renderSubNav(navItemGroup) {
-    const menuItems = navItemGroup.subTags.map(({ node: navItem }, index) => (
+    const menuItems = navItemGroup.items.map((item, index) => (
       <MenuItem dense key={index}>
-        <Link onClick={this.onClose} route={`${this.linkPath(navItem)}`}>
-          <ListItemText primary={navItem.name} />
+        <Link onClick={this.onClose} route={`${this.linkPath(item)}`}>
+          <ListItemText primary={item.navigationItem.data.content[0].value} />
```

Comment:
I just noticed this `item.navigationItem.data.content[0].value`, which isn't what I had in mind when @dancastellon and I originally discussed the nav items development. I didn't catch this reviewing the GraphQL work, but I think we should fix it before merging this.

What it's doing is filtering the array by language. What I thought it would do is set `content` to the actual content for that language. So in the schema `content: [NavigationItemContent]` would be `content: String` instead.

Since we may need the full content array for operator UI, I'm thinking maybe we can leave what's there but add this to `NavigationItemData`:

```
contentForLanguage(language: String!): String
```

If we put the `language` param on that field rather than on the whole `defaultNavigationTree` query, I think it will be simpler and easier to understand what it's for.

Then here you would just do `item.navigationItem.data.contentForLanguage`

Thoughts @dancastellon ?

Reply:
I was a bit confused with this when setting it up. It makes sense to me to either have all the languages sent to the client, or only one, but not in an `array` like we currently have it.

Reply:
@aldeed I just discussed this with @kieckhafer. I think it makes sense to remove the array filtering in the `content` resolver for the operator UI, and add that `contentForLanguage` resolver. I'll make the change.